package.json

```diff
@@ -11,6 +11,7 @@
     "jest": "^29.2.1",
     "lerna": "^5.1.6",
     "ts-jest": "^29.0.3",
+    "tsc-watch": "^4.5.0",
```

Comment:
```suggestion
```

Reply:
Hmm, I must have messed up the merge, fixed.